java/tools/src/java/org/apache/orc/tools/JsonFileDump.java

```diff
@@ -352,6 +353,12 @@ private static void writeColumnStatistics(JSONWriter writer, ColumnStatistics cs
           writer.key("sum").value(((DecimalColumnStatistics) cs).getSum());
         }
         writer.key("type").value(OrcProto.Type.Kind.DECIMAL);
+      } else if (cs instanceof CollectionColumnStatistics) {
+        writer.key("minChildren").value(((CollectionColumnStatistics) cs).getMinimumChildren());
+        writer.key("maxChildren").value(((CollectionColumnStatistics) cs).getMaximumChildren());
+        writer.key("totalChildren").value(((CollectionColumnStatistics) cs).getTotalChildren());
+        /* What about MAP Type ? */
+        writer.key("type").value(OrcProto.Type.Kind.LIST);
```

Comment:
I'd suggest removing the type key rather than putting out a potentially incorrect list.

Reply:
Fixed it.